tests/test_subdomains.py

```diff
@@ -0,0 +1,60 @@
+import numpy as np
+from math import floor
+
+from conftest import skipif
+from devito import Grid, TimeFunction, Eq, solve, Operator, SubDomains
+
+pytestmark = skipif(['yask', 'ops'])
+
+
+class TestSubdomains(object):
+    """
+    Class for testing SubDomains
+    """
+
+    def test_iterate_NDomains(self):
```

Comment:
should we add more tests or do you think this one suffices?

Reply:
No, we need a couple more.